blogposts/structural-code-search-at-scale-from-your-browser.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Structural code search. At scale and from your browser.
+author: Rijnard van Tonder
+authorUrl: https://twitter.com/rvtond
+publishDate: 2020-01-24T10:00-07:00
+tags: [blog]
+slug: structural-code-search-at-scale-from-your-browser
+heroImage: https://TODO
+published: true
+---
+
+We're introducing a richer way to search code at Sourcegraph with structural
+code search. And we're making it available at scale.
```

Comment:
```suggestion
code search.
```

To me, this puts into question whether everything we do is at scale so I'd remove it.

Reply:
As above comment, I think the scale part is important from the direction of 'this is what Sourcegraph enables'. But maybe that's not clear from this sentence so I'm OK to remove it.